src/game.cpp

```diff
@@ -995,14 +995,16 @@ static void updateChat(Client &client, f32 dtime, bool show_debug,
 	// Update gui element size and position
 	s32 chat_y = 5;
 
-	if (show_debug)
-		chat_y += 3 * line_height;
+	if (flags.show_debug) {
+		chat_y += 2 * line_height;
+	} else if (flags.show_status) {
+		chat_y += 1 * line_height;
```

Comment:
Nice, i agree with having a 'show status' flag to determine how far chat is moved down, but i think it should just be an internal variable not settable with a setting.